yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryExecutor.java

```diff
@@ -258,6 +260,10 @@ private Object executeStatementWithForcedContinuations(final @Nonnull Statement
                         Assertions.assertFalse(hasNext);
                     }
                 }
+                count = count + 1; // PMD failure for ++
```

Comment:
```suggestion
                count += 1;
```

Reply:
Yeah. Wouldn't it be nice if PMD allowed for `if (++count < MAX)`?

Reply:
Done.